deployment/reverseProxy/reverseProxy.rst

```diff
@@ -44,3 +44,20 @@ A popular open source alternative is Nginx. For instruction on how to deploy Fir
    IIS <iis>
    Nginx <nginx>
 
+Using X-Forwarded-Host header
+-----------------------------
+
+The `X-Forwarded-Host header <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host>`_ is used for identifying the original host by the client, which is especially useful with reverse proxies. As per the `FHIR specification <https://build.fhir.org/http.html#custom>`_, Firely Server supports usage of this header. Keep in mind that usage of the header exposes privacy sensitive information.
```

Comment:
I wouldn't link to the build version of the spec as it might not always be available

Reply:
Thank you! Done 👍